### PR TITLE
Ali/python interface

### PR DESCRIPTION
**Pull request by alisonsimons**

### CODE REVIEW

File: maxwell_slice/sphere_scat.py [hunk 1]
```diff
@@ -1,10 +1,11 @@
-from ._temporarydirectory import TemporaryDirectory
-from ._shellscript import ShellScript
+from _temporarydirectory import TemporaryDirectory
```
Comment (magland) [inline maxwell_slice/sphere_scat.py:1]: I think we should make an examples folder and put sphere_scat.py as `examples/sphere_scat_example.py`

File: maxwell_slice/sphere_scat.py [hunk 2]
```diff
@@ -16,6 +17,9 @@ def sphere_scat():
         # Here is where we run the fortran program using a bash script
         s = ShellScript(f'''
         #!/bin/bash
```
Comment (magland) [inline maxwell_slice/sphere_scat.py:19]: We need to make sure that we have the proper working directory for this script, in order to find the fortran program.

File: scratchspace/SphereScat/testdriver4f90.f [hunk 3]
```diff
@@ -356,7 +356,9 @@
      1    centers,nterms,sh1,sh2,targ,ntargs,htot,
      2    fld,cw,lcw,lused,ier)
       open(unit=20, file='data.m',status='unknown') 
-      open(unit=21, file='field.m',status='unknown') 
+      open(unit=21, file='field.m',status='unknown')
+      open(unit=22, file='data.bin',status='unknown')
```
Comment (magland) [inline scratchspace/SphereScat/testdriver4f90.f:360]: @alisonsimons, should we contact Leslie to get a hint on how to write the .bin data in fortran?

Comment (alisonsimons) [reply scratchspace/SphereScat/testdriver4f90.f:360]: @magland Yeah seems like a good idea

**Pull request merged**